packages/api/package.json

```diff
@@ -35,7 +35,7 @@
   "scripts": {
     "build": "tsc",
     "build:watch": "tsc --watch",
-    "test": "ts-mocha --recursive=true ./src/libs/**/*.test.ts ./src/tables/**/*.test.ts", 
```

Comment:
curious that this got through the linter in CI

Reply:
No idea :/